tools/cgpatch/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(CGPatch)
+
+# LLVM
+if(NOT LLVM_ENABLE_RTTI)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
```

Comment:
Probably `target_compile_options` instead of the global var.

Reply:
I think setting those global vars should go away, since you added the target specific setting of the options.